net-misc/mstpd/mstpd-0.0.8.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1 systemd
+
+DESCRIPTION="Multiple spanning tree protocol daemon"
+HOMEPAGE="https://github.com/mstpd/mstpd"
+SRC_URI="https://github.com/mstpd/mstpd/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	default
+
+	sed -i -e 's:-Werror::g' Makefile.am || die
```

Comment:
can you tell upstream to make this an option? saves us the (slow) autoreconf

Reply:
Sure, I'll forward it

Reply:
Forwarded in https://github.com/mstpd/mstpd/issues/105